lib/extensions/src/lib/services/app-extension.service.ts

```diff
@@ -31,22 +29,22 @@ export class AppExtensionService {
 
     references$: Observable<ExtensionRef[]>;
 
-    constructor(private extensions: ExtensionService) {
+    constructor(private extensionService: ExtensionService, private logService: LogService) {
```

Comment:
The idea was that Core will reference Extensions once we start moving stuff, not the Extensions referencing Core. Otherwise, we have to carry the huge Core dependency on extension projects.